### ISSUE TITLE
Missing IOTA and now unable to login

### ISSUE THREAD

**Issue by Padmin29** [iotaledger/wallet#1192]
### Prerequisites
If you are **not** suggesting a feature, you must be able to check all of the following (place an x inside the brackets to check the box). If you cannot check all of the boxes, this issue should not be reported on GitHub as it is not a bug:

* [ ] I confirm that this is an issue with the IOTA Wallet and not an exchange
* [ ] I confirm that this is an issue with the wallet, not IRI
* [ ] For pending transactions: I confirm that I have tried reattaching and promoting multiple times (check this box if your issue is not related to a pending transaction)
* [ ] I confirm that this is not an issue related to stolen/lost funds
* [ ] I confirm that this is not a zero balance issue related to a snapshot (last snapshots occurred on October 23 and January 28)
* [ ] For private key warnings: I confirm that I am not at risk of reusing an address, or another user has verified that I am not (check this box if your issue is not related to a private key reuse)
* [ ] I confirm that this issue is present on the latest version of the wallet

### Description

[Description of the bug or feature]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

### Versions

[OS Version]
[Wallet Version]

**Comment by wonkytonky** [comment]
fill in template

**Comment by lo5tinpr0** [comment]
Hey @Padmin29 , which node do you use for the Light Wallet?

**Comment by Padmin29** [comment]
Currently using 'https://nodes.iota.cafe:443' but have tried almost all
nodes in the list. I have also reinstalled the light wallet (Ver.2.5.7)
...confirmed firewall and VPN are not blocking any traffic ...and attempted
to reattach to tangle via Receive option in the wallet

Please advise,

Thanks

**Comment by lo5tinpr0** [comment]
To be clear, you're able to login with your seed? Sound's like you're able to log in and just having the zero value issue.
If you want to, you can post your IOTA address (NOT seed) where your IOTAs should be and I will have a look at the transaction. Sometimes, there are transactions which seems to "stuck" in the pending process.

**Comment by wonkytonky** [comment]
if your seed is correct, reattach a lot .. until you see your balance .. 
and use a good node(synced one)

**Comment by Padmin29** [comment]
Wallet Address:

CKWDZTHYHKH9SE9AU9CKZY9MLT9WWDMQDZCBKFOIIBURQRXYNETFUHPJLTRBPFJZM9BVNJBTNQQRPBFCXZMS9IUN9C

Let me know if you have any luck

Thanks again

**Comment by rajivshah3** [comment]
It doesn't look like there's any transactions associated with that address. Can you provide some information about when you last saw a balance in your wallet? That might help us figure out what the issue is

**Comment by Padmin29** [comment]
I made my purchases on the following dates via Binance and moved to wallet
shortly after each buy and have not moved them since

12/18/2017 48
12/31/2017 13
1/06/2018 45

Balance in wallet after all transfers was 105.4

You guys would know better than me but I believe the wallet address changes
every time you receive ..these are other addresses that I've used

EJGZPE9EYPMLUIPWVPFQOC9STSMWE9OPR9HHEEJZMPDO
AZPGGDKAPVIWOYVWQOAGZMCUNUZDGAKHQMVWDPRYRODJD

EJGZPE9EYPMLUIPWVPFQOC9STSMWE9OPR9HHEEJZMPDOEAZPGGDKAPVIWOYVWQOAGZMCUNUZDGAKHQMVWDPRYRODJD


RV9SGWLFIHPUMTAWJYDZKILJMANCGKHEFX9HLCYIDY9RXTKJMFARRBAYSMYVZ9ILYNDPZSTSPUMGEAJFWBCVPHK9RX

again  ..thanks for the support

**Comment by lo5tinpr0** [comment]
Hey @Padmin29 ,

I had a look into your IOTA addresses. At first, I must mention that this address 

>CKWDZTHYHKH9SE9AU9CKZY9MLT9WWDMQDZCBKFOIIBURQRXYNETFUHPJLTRBPFJZM9BVNJBTNQQRPBFCXZMS9IUN9C

isn't attached to the tangle, which means, that this address doesn't exist on the tangle. I guess, that you created the address, but didn't attached it to the tangle.

Secondly, this address

> EJGZPE9EYPMLUIPWVPFQOC9STSMWE9OPR9HHEEJZMPDO AZPGGDKAPVIWOYVWQOAGZMCUNUZDGAKHQMVWDPRYRODJD

doesn't exists at all. There's a letter missing. See third.

Thirdly, this address 

> EJGZPE9EYPMLUIPWVPFQOC9STSMWE9OPR9HHEEJZMPDOEAZPGGDKAPVIWOYVWQOAGZMCUNUZDGAKHQMVWDPRYRODJD

is the interest one. You can have a look on the address here: [https://thetangle.org/address/EJGZPE9EYPMLUIPWVPFQOC9STSMWE9OPR9HHEEJZMPDOEAZPGGDKAPVIWOYVWQOAGZMCUNUZDGAKHQMVW](url)
On January 8, 2018 22:18:58, there is the confirmed transaction of 105.452 Mi. Everything fine. But on January 23, 2018 10:27:55 all the 105.452 Mi were transferred to that address: [https://thetangle.org/address/XADKVAWYYUXBMRKSPSRASKYBAZGMBUUNC9KISHSAWJRSEQDHXRUDALUNFDFUJXJSJRLPKQPADFY9WGSKW](url)
If you have a look on that address, you can see your 105.452 Mi (at January 23, 2018 10:27:55) and several other transactions. 
On February 3, 2018 16:50:58 the complete amount of this address (516.732 Mi) was transfered to two different addresses. You can see this by following the last bundle link: [https://thetangle.org/bundle/OQEAHUETCVQZBDIKFRFXDIEEJIGVYVGKRBNPFHCETXPHGTFLPOUKLLDDYICGCJFTERBFHDKPWTCRNLAVX](url)
That's the typical operating mode of steeling IOTAs. And no, there is no chance to bring back your IOTAs or find out who's behind it.

For me, it looks like someone else had access to your seed and transfered all your IOTAs to his own address.

Fourthly, this address 

> RV9SGWLFIHPUMTAWJYDZKILJMANCGKHEFX9HLCYIDY9RXTKJMFARRBAYSMYVZ9ILYNDPZSTSPUMGEAJFWBCVPHK9RX

has just zero value transactions. Seems everything ok with that one.

> You guys would know better than me but I believe the wallet address changes every time you receive ..these are other addresses that I've used

You're completely right. For every receive, you have to create a new address. And you have to attach this address to the tangle first. Then, you're able to receive IOTAs to that address.

Back to the seed topic and steeling IOTAs: How did you create your seed? Is there any chance you created it online or that someone else had a chance to get to know your seed? Because from a technical view, everthing worked fine and as it should.

**Comment by wonkytonky** [comment]
fyi. you don't need to attach any address to the tangle  in theory to recieve money.   it's good practice to do so though.

**Comment by lo5tinpr0** [comment]
You're absolutely right... but for the normal "I want to receive some IOTAs" use, it's recommended.

**Comment by rajivshah3** [comment]
@Padmin29 unfortunately, as @lo5tinpr0 said, your funds may have been stolen. The time of the transactions coincides with a period that a lot of IOTA were stolen due to a malicious seed generator. Is it possible that you used a seed generator such as iotaseed.io or iotaseedgenerator.org?

**Comment by Padmin29** [comment]
Well that sucks ...had that gut feeling ...thx for the help guys

**Comment by lo5tinpr0** [comment]
Yep, @Padmin29 nr. 1 rule with IOTA: Never use (online-) seed generators! Never, never, never!

If you want to create a random 81 character string you're free to use so called password managers e.g. KeePass 2. Creating a safe seed and store it properly is the most important thing using IOTA.

All the best for the future. This issue might be closed.

**Comment by kons233** [comment]
hey @lo5tinpr0 , how do you read from what address to what address iota is transferred from and to? Binance is asking me to tell them what my deposit address is on their exchange, but my iota wallet is no longer listing the history transactions since the update. My balance shows but not how many iota i have received or sent and where to and from.

**Comment by lo5tinpr0** [comment]
Hey @kons233 , I made a short video for you, showing how to find the IOTA address where IOTAs were sent to.
(Sry, zero viedo skills at all)
You don't have to download it, you can watch it in your browser :)

[https://mega.nz/#!HOJwTSwa!3vDsIffVIKHG1aLfLNgEfYObthqsWGRQRkLFz9HGngs](url)

The only two things you need are an IOTA address and [https://thetangle.org/](url)
That's it.
In the video, I took 
> Thirdly, this address

> EJGZPE9EYPMLUIPWVPFQOC9STSMWE9OPR9HHEEJZMPDOEAZPGGDKAPVIWOYVWQOAGZMCUNUZDGAKHQMVWDPRYRODJD

from my post above.

About Binance: I'm not familiar with Binance because I use Bitfinex, but I wonder why Binance wants to know a deposit address from you. What's the point?
Normally each deposit address should only be used once. Therefore Binance should give you the deposit address and not ask for one. This is the Bitfinex note about IOTA and deposit addresses: 

![screenshot-2018-4-11 bitfinex](https://user-images.githubusercontent.com/37935152/38592200-fb002e1a-3d3a-11e8-8141-6a67577b9cd5.png)

There is a chance to restore your transaction history. Try to add multiple new receive addresses to the tangle. Sometimes the history of the transactions appears again.

Let me know how you're doing and if I can help you any further.

**Comment by kons233** [comment]
@lo5tinpr0  Thanks so much man, I appreciate you taking the time to do that, the link isn't working on my phone but I'll check it out as soon as I get in front of a computer. I looked up the addresses of @Padmin29  in your post trying to help him figure out where his Iota went, and when I open his address on thetangle.org I see the negative and positive transactions, but on my wallet since the snap shot I no longer see the addresses to transactions, when I open the addresses with money on them they only show dates and large balances, not individual transactions, but ya I'll watch the video first and if it's different I'll dig up one of the addresses to show you. As far as Binance I got a new phone and I assumed that my Google authenticator codes would simply transfer to my new phone since it's under the same account, but that was not the case and I had already wiped the old phone, so I got locked out of every exchange I'm on. So to confirm my identity Binance automatically asked for the last address I deposited in. Thanks again.
